src/Views/Homepage.vala

```diff
@@ -101,19 +99,16 @@ public class AppCenter.Homepage : Hdy.Deck {
             valign = Gtk.Align.START
         };
 
-        var grid = new Gtk.Grid () {
-            column_spacing = 24,
-            orientation = Gtk.Orientation.VERTICAL
-        };
-        grid.add (banner_revealer);
-        grid.add (recently_updated_revealer);
-        grid.add (categories_label);
-        grid.add (category_flow);
+        var box = new Gtk.Box (orientation = Gtk.Orientation.VERTICAL, 0);
```

Comment:
I guess you do not need to reproduce the column_spacing property.

Reply:
Yeah since it's vertical column spacing was unused